schema/playground/health.schema

```diff
@@ -0,0 +1,14 @@
+package playground;
```

Comment:
Why is this being added to playground? Is it needed for testing?

Reply:
testing controls and features on mobile. if it will ever be merged into master is another question. but as long as we don't have a proper starter project for mobile, playground it is

Reply:
Why not make a starter project repo for mobile then?
I'd rather not have a grab bag of features in playground. It will make cleaning it up much harder if nothing else.

Reply:
it's now gonna be moved to a different repo. We haven't really decided on a starter porject yet, so gonna create a proper repo later